app/views/viewings/edit.html.erb

```diff
@@ -1,3 +1,7 @@
-<%= viewing.theatername %>
-<%= viewing.playtitle %>
-<%= viewing.date %>
\ No newline at end of file
+<h1>Edit Viewing</h1>
+
+<%= form_tag('/viewings/@viewing.id/update') do %>
```

Comment:
When you say report back, are you asking me to coming back to OCS, or do you just mean report back via "comment"?